vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -312,6 +276,14 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{Err: errGetDevicePath.Error()}
 	}
 
+	if skipInotify {
+		time.Sleep(sleepBeforeMount)
+	}
+
+	// Wait for the attach to complete, may timeout
+	// in which case we continue creating the file system.
+	fs.DevAttachWait(watcher, r.Name, device)
```

Comment:
it looks like fs.DevAttachWait() is called even if skipNotify is true. Is it by design  ?